src/praisonai/praisonai/sandbox/daytona.py

```
@@ -0,0 +1,464 @@
+                "id": str(uuid.uuid4()),
+                "name": self.workspace_name,
+                "template": self.workspace_template,
+                "provider": self.provider,
+                "status": "running",
+                "endpoint": f"https://{self.workspace_name}.daytona.dev",
+            }
+            
+            self._is_running = True
+            logger.info(f"Daytona workspace created: {self.workspace_name}")
+            
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)